drivers/hwinfo/hwinfo_smartbond.c

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2023 Jerzy Kasenberg.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/device.h>
+#include <zephyr/drivers/hwinfo.h>
+#include <DA1469xAB.h>
+
+int z_impl_hwinfo_get_reset_cause(uint32_t *cause)
+{
+	int ret = 0;
+	uint32_t reason = CRG_TOP->RESET_STAT_REG;
+	uint32_t flags = 0;
+
+	/*
+	 * When POR is detected some other bits are also set that
+	 * are not really valid so they are skipped.
```

Comment:
What dies that mean "are not really valid"?
Is there a chip errata or something like that?

Reply:
Comment reworded, should be more readable now